hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4142,38 +4141,60 @@ func (r *HostedClusterReconciler) validateReleaseImage(ctx context.Context, hc *
 		currentVersion = &version
 	}
 
-	// Validate release image is multi-arch
-	if hc.Spec.Platform.Type == hyperv1.AWSPlatform && hc.Spec.Platform.AWS.MultiArch {
-		isMultiArchReleaseImage, err := registryclient.IsMultiArchManifestList(ctx, hc.Spec.Release.Image, pullSecretBytes)
-		if err != nil {
-			return fmt.Errorf("failed to determine if release image multi-arch: %w", err)
-		}
+	minSupportedVersion := supportedversion.GetMinSupportedVersion(hc)
 
-		if !isMultiArchReleaseImage {
-			return fmt.Errorf("release image is not a multi-arch image")
-		}
+	return supportedversion.IsValidReleaseVersion(&version, currentVersion, &supportedversion.LatestSupportedVersion, &minSupportedVersion, hc.Spec.Networking.NetworkType, hc.Spec.Platform.Type)
+}
+
+// determineHostedClusterPayloadType returns the HostedCluster payload's CPU architecture type
+func (r *HostedClusterReconciler) determineHostedClusterPayloadType(ctx context.Context, hc *hyperv1.HostedCluster, imageMetadataProvider hyperutil.ImageMetadataProvider) (string, error) {
+	pullSecretBytes, err := r.getPullSecretBytes(ctx, hc)
+	if err != nil {
+		return "", err
 	}
 
-	// Validate each NodePool CPU arch matches the management cluster's CPU arch when the Hosted Cluster is not multi-arch
-	if hc.Spec.Platform.Type == hyperv1.AWSPlatform && !hc.Spec.Platform.AWS.MultiArch {
-		nodePoolList := &hyperv1.NodePoolList{}
-		if err := r.List(ctx, nodePoolList, client.InNamespace(hc.Namespace)); err != nil {
-			return fmt.Errorf("failed to get list of nodepools for cpu arch validation: %w", err)
-		}
+	isMultiArchReleaseImage, err := registryclient.IsMultiArchManifestList(ctx, hc.Spec.Release.Image, pullSecretBytes)
+	if err != nil {
+		return "", fmt.Errorf("failed to determine if release image multi-arch: %w", err)
+	}
 
-		mgmtClusterCPUArch := runtime.GOARCH
+	if isMultiArchReleaseImage {
+		return "Multi", nil
+	}
 
-		for _, nodePool := range nodePoolList.Items {
-			if err := hyperutil.DoesMgmtClusterAndNodePoolCPUArchMatch(mgmtClusterCPUArch, nodePool.Spec.Arch); err != nil {
-				return err
-			}
-		}
+	arch, err := getImageArchitecture(ctx, hc.Spec.Release.Image, pullSecretBytes, imageMetadataProvider)
```

Comment:
Just because sometimes we don't have a release image in the CLI and we just have a release stream. In the CLI if it's a multi image or stream we don't query the mgmt cluster version.